storybook/index.html

```diff
@@ -446,23 +468,47 @@ <h1 class="sectionHeading">BAMP</h1>
                     </div>
                 </section>
                 <section class="demo" data-module-name="muralModal">
-                    <div class="muralModal">
-                        <img class="muralModal--img" src="https://greatschoolvoices.org/wp-content/uploads/2020/06/Lodestar1.jpeg">
-                        <div class="muralModal--caption">
-                            <h2 class="muralModal--name">Lodestar</h2>
-                            <button class="muralModal--close" aria-label="Close">×</button>
-                            <p class="muralModal--text">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis quis nunc sed ipsum faucibus finibus. Donec sed efficitur lectus. Fusce risus tellus, sodales vel tellus convallis, aliquam ullamcorper velit. Vestibulum commodo velit ac eleifend tincidunt. Quisque tincidunt purus sem, at interdum dolor hendrerit eu. Quisque dui metus, laoreet at malesuada fringilla, ullamcorper tempus mauris.</p>
-                            <p>
-                                <img class="muralModal--icon" src="../images/palette.svg">
-                                <span class="muralModal--bold">Lead Artist</span><span class="muralModal--text">: Rachel</span>
-                            </p>
-                            <p>
-                              <img class="muralModal--icon" src="../images/map-marker.svg">
-                              <span class="muralModal--bold">Location</span><span class="muralModal--text">: Lodestar Public School</span>
-                            </p>
-                            <div class="muralModal--end">
-                                <img class="muralModal--icon" src="../images/calendar.svg">
-                                <span class="muralModal--bold">Date</span><span class="muralModal--text">: April, 2020</span>
+                    <div id="muralModalContainer" class="muralModalContainer">
```

Comment:
same as above, we can't repeat IDs, and since we know we have multiple modals, we should just not use IDs at all for this.

I would also put the `data-mural-name` on the outer container div, like so:

```HTML
<div class="muralModalContainer" data-mural-name="lodestar">
    <diiv class="muralModal">...</div>
</div>
```

...the reason is that the container is what we'll be hiding and showing, so it will simplify the JS later